cmd/prysmctl/testnet/generate_genesis.go

```diff
@@ -265,6 +285,39 @@ func generateGenesis(ctx context.Context) (*ethpb.BeaconState, error) {
 	if err != nil {
 		return nil, err
 	}
+	if overrideEth1Data {
+		log.Print("Overriding Eth1Data with data from execution client")
+		conn, err := rpc.Dial(generateGenesisStateFlags.ExecutionEndpoint)
+		if err != nil {
+			return nil, errors.Wrapf(
+				err,
+				"could not dial %s please make sure you are running your execution client",
+				generateGenesisStateFlags.ExecutionEndpoint)
+		}
+		client := ethclient.NewClient(conn)
+		header, err := client.HeaderByNumber(ctx, nil)
+		if err != nil {
+			return nil, errors.Wrap(err, "could not get header by number")
+		}
+		depositContract, err := deposit.NewDepositContract(
```

Comment:
For a genesis state, I think we can avoid calling the contract like this. Instead we just assume that it has no deposits. 
```
t, err := trie.NewTrie(params.BeaconConfig().DepositContractTreeDepth)
	if err != nil {
		return [32]byte{}, err
	}
		genesisState.Eth1Data = &ethpb.Eth1Data{
			DepositRoot:  t.HashTreeRoot()[:],
			DepositCount: 0,
			BlockHash:    header.Hash().Bytes(),
		}
genesisState.Eth1DepositIndex = 0